packages/pilot/src/pages/ErrorBoundary/ErrorBoundary.js

```diff
@@ -0,0 +1,146 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+import {
+  applyTo,
+  both,
+  compose,
+  complement,
+  equals,
+  evolve,
+  find,
+  identity,
+  ifElse,
+  is,
+  isNil,
+  pipe,
+  prop,
+  propEq,
+  propSatisfies,
+  uncurryN,
+} from 'ramda'
+import { withRouter } from 'react-router-dom'
+import { translate } from 'react-i18next'
+
+import { receiveReactError } from '.'
+
+const mapStateToProps = ({ errors }) => ({ errors })
+
+const mapDispatchToProps = dispatch => ({
+  receiveError: (error, stack) => {
+    dispatch(receiveReactError({ error, stack }))
+  },
+})
+
+const enhanced = compose(
+  connect(
+    mapStateToProps,
+    mapDispatchToProps
+  ),
+  withRouter,
+  translate()
+)
+
+const findInErrorsRoutes = uncurryN(2, route => find(pipe(
```

Comment:
Separando este pipe em uma função fica mais clara a leitura
```suggestion
const isAffectedRoute = route => pipe(
  prop('affectedRoute'),
  equals(route)
)
const findInErrorsRoutes = uncurryN(2, route => find(isAffectedRoute(route)))
```